src/_P028_BME280.ino

```diff
@@ -281,24 +304,24 @@ boolean Plugin_028(uint8_t function, struct EventStruct *event, String& string)
             String log;
 
             if (log.reserve(40)) { // Prevent re-allocation
-              log  = P028_data->getDeviceName();
+              log  = P028_data->getDeviceName(P028_data->sensorID);
               log += F(": Address: ");
               log += formatToHex(P028_I2C_ADDRESS, 2);
               addLogMove(LOG_LEVEL_INFO, log);
 
               // addLogMove does also clear the string.
-              log  = P028_data->getDeviceName();
+              log  = P028_data->getDeviceName(P028_data->sensorID);
               log += F(": Temperature: ");
               log += formatUserVarNoCheck(event->TaskIndex, 0);
               addLogMove(LOG_LEVEL_INFO, log);
 
               if (P028_data->hasHumidity()) {
-                log  = P028_data->getDeviceName();
+                log  = P028_data->getDeviceName(P028_data->sensorID);
```

Comment:
Strictly speaking, with static functions, you don't need to address them as member functions.
`P028_data_struct::getDeviceName(P028_data->sensorID);` would make it more clear it is a static function and not a member function.